### PR TITLE
Add code examples of adding callbacks on the collapsible widget events

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
## Purpose of this pull request

This pull request adds code examples of adding callbacks on the collapsible widget events.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_collapsible.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_collapsible.html
- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_collapsible.html

## Links to Magento source code

*Magento 2.3:*
- https://github.com/magento/magento2/blob/2.3/lib/web/mage/collapsible.js#L444
- https://github.com/magento/magento2/blob/2.3/lib/web/mage/collapsible.js#L481 
- https://github.com/magento/magento2/blob/2.3/lib/web/mage/collapsible.js#L519 

*Magento 2.2:* 
- https://github.com/magento/magento2/blob/2.2/lib/web/mage/collapsible.js#L438
- https://github.com/magento/magento2/blob/2.2/lib/web/mage/collapsible.js#L476
- https://github.com/magento/magento2/blob/2.2/lib/web/mage/collapsible.js#L514

*Magento 2.1:* 
- https://github.com/magento/magento2/blob/2.1/lib/web/mage/collapsible.js#L357 
- https://github.com/magento/magento2/blob/2.1/lib/web/mage/collapsible.js#L380 
- https://github.com/magento/magento2/blob/2.1/lib/web/mage/collapsible.js#L414 

whatsnew
Added new code examples showing you [how to use callbacks](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_collapsible.html) on collapsible widget events.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by serhiyzhovnir** [comment]
Hi @ronak2ram 
The task description was updated.
Please, check this PR again.
Thank you!

File: guides/v2.1/javascript-dev-guide/widgets/widget_collapsible.md [hunk 1]
```diff
@@ -513,9 +513,32 @@ $("#element").collapsible("forceDeactivate");
 ### `beforeOpen callback` {#c_beforeOpen}
 Called before the content is opened. 
 
+Example of adding callback on `beforeOpen` event:
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_collapsible.md]:
```suggestion
Example of adding a callback to `beforeOpen` events:
```

File: guides/v2.1/javascript-dev-guide/widgets/widget_collapsible.md [hunk 2]
```diff
@@ -513,9 +513,32 @@ $("#element").collapsible("forceDeactivate");
...
+```javascript
+$("#element").on("beforeOpen", function () {
+    // do something before opening the content
+});
+```
+
 ### `dimensionsChanged` {#c_dimensionsChanged}
 Called after content is opened or closed.
 
+Example of adding callback on `dimensionsChanged` event:
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_collapsible.md]:
```suggestion
Example of adding a callback to `dimensionsChanged` events:
```

**Comment by serhiyzhovnir** [comment]
Hi @jeff-matthews,
The suggested changes were added.
Thank you!

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**